src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.cs

```diff
@@ -46,6 +46,8 @@ private struct CacheLineSeparated
         private static int s_completionCount = 0;
         private static int s_threadAdjustmentInterval;
 
+        private static volatile int s_numRequestedWorkers = 0;
+
         static ClrThreadPool()
```

Comment:
Ok. I'll try to do the initialization inline. I have one question: can I rely on the static members being initialized in the order they are specified in the source file?

If not, I need to use either a static constructor or an `Initialize` function with an `EnsureInitialize` call at the start of each static method.